airflow/providers/google/cloud/operators/gcs.py

```diff
@@ -625,3 +625,32 @@ def execute(self, context: Dict):
                 object_name=self.destination_object,
                 filename=destination_file.name
             )
+
+
+class GCSDeleteBucketOperator(BaseOperator):
+    """
+    Deletes bucket from a Google Cloud Storage.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GCSDeleteBucketOperator`
+
+    :param bucket_name: name of the bucket which will be deleted
+    :type bucket_name: str
+    """
+
+    template_fields = ('bucket_name', "gcp_conn_id")
+
+    @apply_defaults
+    def __init__(self,
+                 bucket_name: str,
```

Comment:
```suggestion
                 bucket_name: str,
                 force: bool = True,
```
It's good idea to add this option to operator. In my opinion it should default to `True`.

Reply:
I agree, thank you.